test/WebJobs.Script.Tests.Shared/TestHelpers.cs

```diff
@@ -25,6 +25,7 @@
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Options;
 using Moq;
+using Newtonsoft.Json.Linq;
```

Comment:
What prompted this change?

Reply:
I removed this but must have accidentally come back during the rebase - thanks for catching!